src/pptrees/modules.py

```diff
@@ -198,6 +304,34 @@
 endmodule
 """
 
+buffer_node['vhdl']="""
+entity buffer_node is
+\tport (
+\t\tpin : in std_logic;
+\t\tpout : out std_logic;
+\t\tgin : in std_logic;
+\t\tgout : out std_logic;
+\t);
+end entity;
+
+architecture behavior of buffer_node is
+begin
+
+U1: buffer
+\tport map (
+\t\tA => pin,
+\t\tY => pout
+\t);
+
+U2: buffer
+\tport map (
```

Comment:
Why not just use a tab here (rather than a `\t`)?

Reply:
I've gotten burned inside this code-base before by switching computers and having my differing .vimrc's swap tabs/spaces inside the source code.

I prefer explicitly declaring `\t` to prevent any such text editor overwriting / changing the functionality of my code nonsense.